modules/cmip5.R

```diff
@@ -4,14 +4,22 @@ library(countrycode)
 library(foreach)
 library(stringr)
 
+# if doing a test:
+if ((Test== TRUE) & !dir.exists("data/cmip5/RegionalSCC_rcpfits_Test")){
+  source("modules/generate_test_input.R")
+}
 
-# Load popweighted country temperatureincrease
-files = Sys.glob(file.path("data","cmip5","RegionalSCC_rcpfits","pop*.csv"))
+if (Test == TRUE){
+  # Load for the test files
+  files = Sys.glob(file.path("data","cmip5","RegionalSCC_rcpfits_Test","pop*.csv"))
+}  else{
+  # Load popweighted country temperatureincrease
+  #files = Sys.glob(file.path("data","cmip5","RegionalSCC_rcpfits","pop*.csv"))
```

Comment:
Because I assumed it would either use the files with the basetemp from the test folder or if it is not a test it takes the folder with the origional files